tests/nlu/training_data/converters/test_watson_nlu_json_to_yaml_converter.py

```diff
@@ -0,0 +1,48 @@
+import os
+from pathlib import Path
+
+import pytest
+from rasa.shared.utils.io import read_json_file
+from rasa.nlu.training_data.converters.watson_nlu_json_to_yaml_converter import (
+    WatsonTrainingDataConverter,
+)
+from rasa.shared.constants import LATEST_TRAINING_DATA_FORMAT_VERSION
+import yaml
+
+
+def test_filter():
+    source = Path("data/examples/watson/demo_watson_v2.json")
+    filter = WatsonTrainingDataConverter().filter(source)
+    if filter:
+        assert True
```

Comment:
We should always assert it and not only if we already checked that it's succeeding 
```suggestion
    assert True
```